src/components/Template.js

```diff
@@ -37,12 +35,10 @@ class Template extends React.Component {
       return null;
     }
 
-    const otherProps = omit(this.props, keys(Template.propTypes));
-
     if (React.isValidElement(content)) {
       return (
         <div
-          {...otherProps}
```

Comment:
This is where data={} was sent. In dev it was ok because we did omit using the propTypes, but in production it was not